src/dbs/mod.rs

```diff
@@ -27,6 +27,14 @@ pub struct StartCommandDbsOptions {
 	#[arg(env = "SURREAL_AUTH", long = "auth")]
 	#[arg(default_value_t = false)]
 	auth_enabled: bool,
+	// TODO(gguillemas): Remove this argument once the legacy authentication is deprecated in v2.0.0
+	#[arg(
+		help = "Whether to enable explicit authentication level selection",
+		help_heading = "Authentication"
+	)]
+	#[arg(env = "SURREAL_AUTH_LEVEL_ENABLED", long = "auth-level-enabled")]
```

Comment:
what about `auth-level`? It seems implicit that `true` == `enable` and `false`==`disable`

Reply:
totally forgot about it. I guess we could make it work because they belong to different subcommands, but it may be confusing from a CLI perspective that the same arg accepts different values depending on the subcommand.

Let's keep it this way unless someone has a strong opinion either way.